be/src/storage/tablet.h

```diff
@@ -313,6 +315,7 @@ class Tablet : public BaseTablet {
     // compaction related
     std::unique_ptr<CompactionContext> _compaction_context;
     std::shared_ptr<CompactionTask> _compaction_task;
+    bool _disable_compaction = false;
```

Comment:
```suggestion
    bool _enable_compaction = true;
```
Better to use positive sentence.